### ISSUE TITLE
Eratic Jitter caused when I updated

### ISSUE THREAD

**Issue by Pop000100** [DaymareOn/hdtSMP64#95]
**Describe the bug**
Bones are bouncing for a bit then rocketing off(extreme stretching) then resetting at random.

**To Reproduce**
Happens to most people just standing around in town.

**Expected behavior**
Normal movement.

**Screenshots and videos**
Very big thigh
![eratic behavior](https://user-images.githubusercontent.com/20156032/228362031-58aa8c80-1a37-487e-aa75-d84a5ff534e3.png)

**Desktop (please complete the following information):**
Windows 10

** FSMP**
- Installed version
- CUDA: No_CUDA
- AVX: AVX2

**SKSE**
- installed version
2.2.3

**Skyrim**
1.6.640

**Required files**
[AttachedFiles.zip](https://github.com/DaymareOn/hdtSMP64/files/11094029/AttachedFiles.zip)

**Additional context**
This is behavior that appeared when I updated from 1.49.3 to 1.50.4rc-1

**Comment by DaymareOn** [comment]
Hey Pop,

thank you for the good bug report.
Could you test with the No-CUDA, **AVX** version, please?

Day

**Comment by Pop000100** [comment]
Found the issue my headache clearly prevented me from thinking clearly when I tried to debug this before I opened this.

The base xml from bhunp(which I based mine on), has a setting that only seems to matter in the new version.

<origin x="0" y="0" z="4"/>
setting 4 to 0 solves it.

now that is taken care of may I ask question to someone who might know about ranges

on constraint definition
<linearDamping x="1" y="1" z="1"/>
<angularDamping x="0" y="0" z="0"/>

Is that damping also like the bones 0.0-1.0, a lot of xml's use greater then 1.
